ChangeLog

```diff
@@ -30,6 +30,7 @@ ver. 0.9.5 (2016/XX/XXX) - wanna-be-released
    * New Actions:
      - action.d/firewallcmd-rich-rules and action.d/firewallcmd-rich-logging (gh-1367)
 - Enhancements:
+   * extremely speedup of all sqlite database operations (gh-1436)
```

Comment:
>      Shouldn't it be an option in fail2ban.conf so if it doesn't work for
>      them they could quickly disable it?
>    I'll do that in 0.10 together with any other options, that I promised (and
>    then cherry pick back this options to 0.9)
IMHO this should be atomic with this change/PR.  awaiting on it in 0.10
and then cherry-picking sounds like more work for no benefit.